share/lib/python/neuron/doc.py

```diff
@@ -70,7 +71,7 @@ def doc_asstring(thing, title="Python Library Documentation: %s", forceload=0):
             # document its available methods instead of its value.
             object = type(object)
             desc += " object"
-        return title % desc + "\n\n" + pydoc.text.document(object, name)
+        return title.format(desc) + "\n\n" + pydoc.text.document(object, name)
```

Comment:
I actually think this is wrong here; should stay with % because .format has its own specifier rules. Could maybe modify `title` but we didn't.